proto/viam/app/slammapping/v1/slammapping.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package viam.app.slammapping.v1;
+
+import "app/data/v1/data.proto";
+import "common/v1/common.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "go.viam.com/api/app/slammapping/v1";
+
+service CloudSLAMService {
+    rpc StartMappingSession(StartMappingSessionRequest) returns (StartMappingSessionResponse);
+    rpc GetActiveMappingSessionsForRobot(GetActiveMappingSessionsForRobotRequest) returns (GetActiveMappingSessionsForRobotResponse);
+    rpc GetMappingSessionPointCloud(GetMappingSessionPointCloudRequest) returns (GetMappingSessionPointCloudResponse);
+    rpc ListMappingSessions(ListMappingSessionsRequest) returns (ListMappingSessionsResponse);
+    rpc StopMappingSession(StopMappingSessionRequest) returns (StopMappingSessionResponse);
+}
+
+// StartMappingSession
```

Comment:
I like this pattern - can we use it to separate the types for all of the api methods?

Reply:
yes, i think that would be helpful! just updated the rest